aim/sdk/repo.py

```diff
@@ -739,3 +741,11 @@ def close(self):
     @property
     def is_remote_repo(self):
         return self._client is not None
+
+    @contextmanager
+    def atomic_track(self):
```

Comment:
maybe move to `Run`?

Reply:
Not sure if it is a good idea, because in that case we have to make `client` "public" in `repo` class. Or provide almost the same interface in repo as well.